forge-game/src/main/java/forge/game/ability/effects/MillEffect.java

```diff
@@ -82,6 +83,11 @@ public void resolve(SpellAbility sa) {
                 if (sa.hasParam("Imprint")) {
                     source.addImprintedCards(milled);
                 }
+                // MilledAll trigger
```

Comment:
There's also CostMill

so this should better be moved to Player.mill

Reply:
Oh ha ha I just had branch issues - this is where that comment was